src/utils/axios.js

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios'
+export const publicInstance = axios.create({
+  baseURL: '/api/',
+  timeout: 1000,
+  headers: {'Accept': '*/*'}
+});
+publicInstance.interceptors.request.use(request => {
+  console.log("request",request)
```

Comment:
also you can remove the console.log if not needed

Reply:
that is required to see the request and response. Will remove it after all API integrations